src/test/java/de/retest/recheck/review/ignore/DeletedFilterTest.java

```diff
@@ -0,0 +1,60 @@
+package de.retest.recheck.review.ignore;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import de.retest.recheck.ignore.AllMatchFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.ignore.Filter.ChangeType;
+import de.retest.recheck.review.ignore.DeletedFilter.DeletedFilterLoader;
+import de.retest.recheck.review.ignore.io.Loaders;
+import de.retest.recheck.ui.descriptors.Element;
+
+class DeletedFilterTest {
+
+	@Test
+	void should_filter_deleted_elements() {
+		DeletedFilter cut = new DeletedFilter();
+		Element element = Mockito.mock( Element.class );
```

Comment:
Statically import mock

Reply:
This is a welcome improvement, but not mandatory for an approval, right?